nrfx/hal/nrf_nvmc.h

```diff
@@ -133,6 +134,44 @@ NRF_STATIC_INLINE void nrf_nvmc_nonsecure_mode_set(NRF_NVMC_Type *    p_reg,
                                                    nrf_nvmc_ns_mode_t mode);
 #endif
 
+/**
+ * @brief Function for writing a word into the flash
+ *
+ * Before calling this function, the caller must ensure that:
+ *  * The address is word aligned
+ *  * Write mode is enabled and
+ *  * The flash is ready to accept another write
+ *
+ * Users are recommended to use the NVMC driver
+ * nrfx_nvmc_word_write() instead.
+ *
+ * @param[in] addr  Address of the word to write
```

Comment:
I suggest to use `address` instead of `addr`, as we have in nrfx_nvmc : 
```
 * @param[in] address Address to write to.
 * @param[in] value   Value to write.
```